src/Main/Tasks.cpp

```diff
@@ -120,6 +149,28 @@ void FOEDAG::TclArgs_setSynthesisOptions(const QString& argsStr) {
   }
 };
 
+QString FOEDAG::TclArgs_getPlacementOptions() {
+  // Collect placement Tcl Params
+  QString tclOptions =
+      QString::fromStdString(GlobalSession->GetCompiler()->PlaceMoreOpt());
+  tclOptions += " -" + QString(PLACE_ARG) + " " +
+                pinOptToStr(GlobalSession->GetCompiler()->PinAssignOpts());
+  return tclOptions;
+}
+
+void FOEDAG::TclArgs_setPlacementOptions(const QString& argsStr) {
+  auto [pinArg, moreOpts] = separateArg(PLACE_ARG, argsStr);
+  FOEDAG::Compiler* compiler = GlobalSession->GetCompiler();
+  if (compiler) {
+    QStringList tokens = pinArg.split(" ");
+    if (tokens.count() > 1) {
+      int opt = (int)pinStrToOpt(tokens[1]);
```

Comment:
unused code

Reply:
It's passing the args to PinAssignOpts fn.

Reply:
it is doing nothing after all. This call doesn't perform anything and has no results

Reply:
oh yeah, you are right, opt var is unused.